shared/actions/settings.js

```diff
@@ -176,29 +177,46 @@ function* saveNotificationsSaga(): SagaGenerator<any, any> {
     }
 
     let JSONPayload = []
+    let chatGlobalArg = {}
     for (const groupName in current.groups) {
       const group = current.groups[groupName]
-      for (const key in group.settings) {
-        const setting = group.settings[key]
+      if (groupName === Constants.securityGroup) {
+        // Special case this since it will go to chat settings endpoint
+        for (const key in group.settings) {
+          const setting = group.settings[key]
+          chatGlobalArg[`${ChatTypes.CommonGlobalAppNotificationSetting[setting.name]}`] = setting.subscribed
```

Comment:
little worried about this being this general. don't we just want to handle the single key we're showing ? (ChatTypes.CommonGlobalAppNotificationSetting.plaintextmobile)